WORKSPACE

```diff
@@ -251,9 +251,14 @@ googletest_deps()
 
 http_archive(
     name = "com_google_protobuf",
-    sha256 = "e32100a8013870d24ffc37dad6781a61e5d0c99501bcb04d39c340a1c44a8e63",
-    strip_prefix = "protobuf-26.0",
-    urls = ["https://github.com/protocolbuffers/protobuf/releases/download/v26.0/protobuf-26.0.tar.gz"],
+    patches = [
+        "//buildpatches:com_google_protobuf_18241.patch",
+        "//buildpatches:com_google_protobuf_18242.patch",
+        "//buildpatches:com_google_protobuf_18243.patch",
+    ],
+    sha256 = "3b8bf6e96499a744bd014c60b58f797715a758093abf859f1d902194b8e1f8c9",
```

Comment:
Just FYI, you can also use `integrity =` for a more consistent setup with bzlmod.